Enhance readability regarding Slack info in Readme.md

seokho-son opened the pull request:
In https://github.com/kubernetes/website/blob/main/README.md#get-involved-with-sig-docs ,

There is a confusing information regarding Slack.

```
Slack Get an invite for this Slack
```

I think parentheses are omitted.

So, I added parentheses for `get an invite for this Slack` :)

divya-mohan0209 reviewed (changes requested):
This decreases readability, IMO. Can we have the slack invite as an indented bullet under the main Slack bullet?

seokho-son commented:
@divya-mohan0209 Thank you for the review :)

I updated this PR according to your opinion. (I squashed commits)

```
- [Slack](https://kubernetes.slack.com/messages/sig-docs)
  - [Get an invite for this Slack](https://slack.k8s.io/)
- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
```

- [Slack](https://kubernetes.slack.com/messages/sig-docs)
  - [Get an invite for this Slack](https://slack.k8s.io/)
- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)

divya-mohan0209 commented:
/lgtm
Will wait on another sig docs reviewer to provide the sign-off.
Thank you so much :)

seokho-son commented:
/retitle Enhance readability regarding Slack info in Readme.md

reylejano commented:
The indented bullet under the main Slack link is an improvement
/approve
